core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -168,18 +168,18 @@ public final void success(CoreEvent event) {
   @Override
   public final Publisher<Void> error(Throwable throwable) {
     if (isResponseDone()) {
-      if (LOGGER.isDebugEnabled()) {
+      if (debugLogEnabled) {
         LOGGER.debug(this + " error response was already completed, ignoring.");
       }
       return empty();
     }
 
-    if (LOGGER.isDebugEnabled()) {
+    if (debugLogEnabled) {
       LOGGER.debug(this + " responseDone completed with error.");
     }
 
     if (throwable instanceof MessagingException) {
-      if (LOGGER.isDebugEnabled()) {
+      if (debugLogEnabled) {
         LOGGER.debug(this + " handling messaging exception.");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use the built-in formatting to construct this argument. ([squid:S2629](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2629))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLOTwXotIXnmsUkh17&open=AWkLOTwXotIXnmsUkh17)

[ISSUE_KEY]: AWkLOTwXotIXnmsUkh17